tools/harness-thci/OpenThread_WpanCtl.py

```diff
@@ -2031,9 +2007,8 @@ def scanJoiner(self, xEUI='*', strPSKd='threadjpaketest'):
         if not self.isActiveCommissioner:
             self.startCollapsedCommissioner()
         if self.__sendCommand(cmd)[0] != 'Fail':
-            if self.logThreadStatus == self.logStatus['stop']:
-                self.logThread = ThreadRunner.run(target=self.__readCommissioningLogs, args=(120,))
-            return True
+            if self.__getJoinerState(self):
```

Comment:
Here scanJoiner() is an API only for Commissioner. __getJoinerState() should only be used for checking state on Joiner, but not on Commissioner.